f21/ps.html

```diff
@@ -1,1790 +1,1872 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
Leo it appears you deleted the full file and repasted it, so now I cannot merge it because you simply edited the whole thing and others edited their paragraphs in the meantime. This is not going to work because I cannot merge only your edits into the current upstream version. 
Please refetch ps.html from the upstream, then edit ONLY the paragraphs for the programs assigned to you. Thanks!